src/psyclone/tests/psyir/symbols/symboltable_test.py

```diff
@@ -373,8 +553,11 @@ def test_lookup():
         str(error.value)
 
 
-def test_lookup_visibility_filter():
-    ''' Test the filtering functionality of the lookup() method. '''
+def test_lookup_2():
```

Comment:
I prefer meaningful names for my tests but possibly it's a matter of taste. We could put it to the vote?

Reply:
We can vote on it if you like. I expect I will lose this one :-). One question is whether we all have to conform to the same rules with test names I suppose.

Reply:
I'm not sure I care enough to have the debate :-) Things just get a bit messy when deleting/moving tests.